frontend/catalyst/jax_tracer.py

```diff
@@ -687,7 +687,7 @@ def is_measurement(op):
 
     def is_midcircuit_measurement(op):
         """Only to avoid 100 character per line limit."""
-        return isinstance(op, catalyst.pennylane_extensions.MidCircuitMeasure)
+        return isinstance(op, catalyst.api_extensions.quantum_operators.MidCircuitMeasure)
```

Comment:
Minor: Do we want to keep something similar to the old path as well? `catalyst.api_extensions.MidCircuitMeasure`

Reply:
Hmm, the way I set it up is that `api_extensions` only contains the public exports like the function `measure`, but not the class objects themselves. That way the top-level `__init__` can simply re-export everything in the `api_extensions` namespace.

Now we could change it so the `api_extensions` namespace contains internally used objects as well, and the top-level `__init__` would then selectively re-export the public members of the `api_extensions` namespace (like it did before). 

Happy to go with either :)

Reply:
I would go with the second one, but feel free to keep it like that :+1:

Reply:
No problem, I'll make the change :)

Reply:
Done: https://github.com/PennyLaneAI/catalyst/pull/681/commits/9f41507f1eccf35a6dce739e21e893cbf213a845